tools/c7n_mailer/README.md

```diff
@@ -136,24 +136,26 @@ policies:
           - slack://owners
           - slack://foo@bar.com
           - slack://#custodian-test
+          - slack://webhook/#c7n-webhook-test
         transport:
           type: sqs
           queue: https://sqs.us-east-1.amazonaws.com/1234567890/c7n-mailer-test
 ```
 
 Slack messages support use of a unique template field specified by `slack_template`. This field is unique and usage will not break
 existing functionality for messages also specifying an email template in the `template` field. This field is optional, however,
-and if not specified, the mailer will use the default value `slack_default`. 
+and if not specified, the mailer will use the default value `slack_default`.
 
 Slack integration for the mailer supports three flavors of messaging, listed below. These are not mutually exclusive and any combination of the types can be used.
- 
+
 | Required? | Key                  | Type             | Notes                               |
 |:---------:|:---------------------|:-----------------|:------------------------------------|
 |           | `slack://owners`          | string      | Send to the recipient list generated within email delivery logic |
 |           | `slack://foo@bar.com`     | string      | Send to the recipient specified by email address foo@bar.com |
-|           | `slack://#custodian-test` | string      | Send to the Slack channel indicated in string, i.e. #custodian-test | 
- 
- 
+|           | `slack://#custodian-test` | string      | Send to the Slack channel indicated in string, i.e. #custodian-test |
+|           | `slack://webhook/#c7n-webhook-test` | string      | Send to a Slack webhook; appended with the target channel. |
```

Comment:
I agree with Kapil's earlier comment on a previous PR, the webhook itself should be in the config and there should be documentation stating which credentials (api key or webhook) is used by default. the interface for the user should remain the same, i.e. slack://#channel

Reply:
In this form, a user can specify multiple optional webhooks, but if it's in the config file we'd be limited to no more than one. The Slack token only enables support for the other Slack notify mechanisms but I can make that distinction clearer in the README.

Reply:
I see your point with that, but specifying a webhook still seems like it should still be a separate field, the `to` field refers to the recipient, be it user, email, datadog metric endpoint, or channel. The webhook itself is essentially an authentication method, basically a signed url for slack message transport, moving the webhook into the `to` field introduces an inconsistent interface for the user imo